src/Kuzzle.js

```diff
@@ -1,5 +1,6 @@
 var
   uuid = require('uuid'),
+  EventEmitter = require('./eventEmitter'),
```

Comment:
The PR was still under progress.

You can review it now with the latest updates ;)